specification.md

```diff
@@ -4111,7 +4110,7 @@ For more information, see the [Workflow Error Handling](#Workflow-Error-Handling
 | --- | --- | --- | --- |
 | name | Unique retry strategy name | string | yes |
 | delay | Time delay between retry attempts (ISO 8601 duration format) | string | no |
-| maxAttempts | Maximum number of retry attempts. Value of 0 means no retries are performed | string or number | no |
+| maxAttempts | Maximum number of retry attempts. A positive integer value | string or number | no |
```

Comment:
The issue here is "0", should be "1". 
Setting maxAttempts to 1 means no retries.

Reply:
@tsurdilo I understand the number of retry attempts as a number of times we actually retry. So 1 means we will retry once if the invocation originally failed. Or am I missing something?

Reply:
@tsurdilo then I suggest to name it `maxInvocations` or similar before v1.0 of spec as otherwise it is quite misleading.

Reply:
From the retries perspective, first invocation of an action is an attempt.

Reply:
From the spec:

> The `maxAttempts` property determines the maximum number of retry attempts allowed and is a positive integer value.

But I got you. I will change it to "maximum number of attempts allowed..." so it is more clear and will handle 1 correctly, wdyt?

Reply:
I think all we need to say is "setting this value to 1 disables retries." Maybe that would be the clearest?

Reply:
Fixed.